include/openenclave/bits/report.h

```diff
@@ -205,6 +206,20 @@ typedef struct _oe_policy
     size_t policy_size;
 } oe_policy_t;
 
+/**
+ * Get the signer ID for the specified public key.
+ *
+ * @param pem The signer's public key in PEM format.
+ * @param[out] signer_id A buffer of size OE_SIGNER_ID_SIZE that receives the
+ * signer ID.
+ *
+ * @retval OE_OK upon success
+ * @retval OE_INVALID_PARAMETER At least one parameter is invalid.
+ */
+oe_result_t oe_get_signer_id_from_public_key(
```

Comment:
```suggestion
oe_result_t oe_sgx_get_signer_id_from_public_key(
```

This API appears to be SGX-specific, so put sgx in the function name.